classes/utils/Logger.class.php

```diff
@@ -324,6 +331,13 @@ public static function log($level, $message) {
                 if(self::$levels[$level] > $max) continue;
             }
             
+            // Build final message ...
+            if ($facility['output'] == 'json') {
+                $message = json_encode($messageArray);
+            } else { //text
+                $message = (array_key_exists('user', $messageArray)?'[user '.$messageArray['user'].'] ':'').'['.$messageArray['process'].':'.$messageArray['level'].'] '.$message;
```

Comment:
It might be better to ensure a user key exists and is blank? This one line reads like a bit of a nightmare.